HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,736 @@
+
+![Crypto Transfer Transaction](../assets/hip-904/frictionless-airdrop-cryptoTransferTransaction.png)
+
+### Airdrops
+
+Airdrops are similar to crypto transfers, but differ in one fundamental way - when an airdrop is performed and the receiver does not have available or automatic association slots, rather than failing, the transfer will be kept in state as a *pending transfer*. To represent this new semantic, there will be a new `TokenAirdropTransaction` meant for use specifically by those attempting an airdrop, with the following semantics:
+
+![Token Airdrop Transaction](../assets/hip-904/frictionless-airdrop-tokenAirdropTransaction.png)
+
+> ⚠️ `TokenAirdropTransaction` from accounts will not support tokens with custom fallback fees that require the receiver to pay the fallback fee. An exception exists for treasury accounts for which custom fees are not assessed and thereofre fallback fees are not applicable.
+
+
+When a sender airdrops a token, if the receiver’s account is already associated with the token, then it is effectively treated as a normal crypto transfer, except that an airdrop transaction fee is assessed. This fee exists for all uses of `TokenAirdropTransaction` as a spam-deterrent fee.
+
+If the receiver’s account is **not** associated with the token, but has available auto-association slots, then one of the slots is taken (i.e. `used_auto_associations` is incremented) and the transaction payer will pay for the association fee. They also pay a fee representing the price of a full auto-renewal period’s rent. This is done to make sure the receiver has time to dissociate from the token before renewal takes place. For example, if the airdrop were to happen the minute before account renewal, the token would not included in renewal because it has already been pre-paid by the sender for one full renewal period. Thus, the receiver always has at least one full renewal period to dissociate from any tokens they do not want to own.
+
+If the receiver’s account is **not** associated with the token, and they have **no** free auto-association slots, then the airdrop transfer neither succeeds nor fails.
+
+In this case a *pending transfer* is created and held in state. Tokens in the pending transfer are not deducted from the sender, nor are they credited to the receiver, until the receiver *claims* the airdrop. If the receiver never claims the airdrop, then the sender continues to pay for it forever, or until their account expires, is deleted, or the sender *cancels* the pending transfer.
+
+All pending transfers sent by an account must be canceled before the account can be deleted.
```

Comment:
Typo, canceled -> cancelled

Reply:
Both spellings are correct.  The former is more common in American English while the latter is more common in British English.
Merriam-Webster has [an article](https://www.merriam-webster.com/grammar/canceled-or-cancelled) on the topic.

Reply:
That's interesting! Thank you for the explanation 🙌